doc/quickstart.md

```diff
@@ -1,22 +1,47 @@
 # Quickstart
 
-## Dependencies
+## Requirements
 
 - make
 - kubectl
-- clouds.yaml
-- environment.tfvars
+- terraform
+- yq v2
 
-## Steps
+## Prepare the environment
 
-- `make create`
-- `make kubeconfig`
-- `kubectl --kubeconfig testconfig.kubeconfig get nodes`
-- `make clean`
+You need access to an OpenStack cluster.
+Copy the default environment and adjust the options according to your cloud.
 
-## ToDo
+```
+cp terraform/environments/environment-{default,<YOURCLOUD>}.tfvars
+```
 
-- ENVIRONMENT
-- OS_CLOUD
+Edit `terraform/environments/environment-<YOURCLOUD>.tfvars` with your favourite text editor. Every option without a default value must be set.
+Add a [clouds.yaml](https://docs.openstack.org/python-openstackclient/latest/configuration/index.html#configuration-files) inside of the `terraform` dir, in `~/.config/openstack` or `/etc/openstack`.
+
+It is recommended to set the name of the cloud in the `clouds.yml` to the same value as the `cloud_provider` in the `environment-<YOURCLOUD>.tfvars`, then you only have to specify the `ENVIRONMENT` variable.
+
+## Create a testcluster
+
+```
+# Set the ENVIRONMENT to the value of `cloud_provider`
+export ENVIRONMENT=<YOURCLOUD>`
+
+# Create your environment. This includes a management node as virtual machine
+# in your OpenStack environment as well as a Kubernetes testcluster.
+make create
+
+# Get the kubeconfig of the testcluster
+make get-kubeconfig
+
+# Interact with the testcluster
+kubectl --kubeconfig testcluster.kubeconfig get nodes
+```
+
+## Teardown
+
+```
+make clean
```

Comment:
Maybe add a link to teardown.md

It usually ends with an error and fullclean is needed

Reply:
Hmmm, the standard quick-start scenario where you create just the testcluster should normally work with the standard make clean.

Reply:
Sorry, you are right it works. I just messed up something.